README.md

```diff
@@ -2,10 +2,10 @@
 I developed a REST API using Express, by creating a proper series of endpoints that perform CRUD operations on a database, using the REST standard.
 
 ## Deployed Heroku
-- 
+- https://basic-api-server.herokuapp.com/
 
 ## Github Pull Request(s)
-- 
+- https://github.com/nickdeans/basic-api-server/pull/2
 
 ## UML 
```

Comment:
Something else that I would love to see added to the README would be what information someone has to add locally to an env file!